fairing/builders/knative/models/build.py

```diff
@@ -91,19 +87,15 @@ def wait_for_build_completion(self):
         timeout = 180
         check_start = datetime.datetime.now()
         while True:
-            try:
-                bld = self._api_custom.get_namespaced_custom_object(self._group,
-                                                                self._api_version,
-                                                                self._metadata.namespace,
-                                                                self._plural,
-                                                                self._metadata.name)
-            except ApiException as e:
-                logger.error(
-                    "Exception when calling CustomObjectsApi->get_namespaced_custom_object: %s\n" % e)
+            bld = self._api_custom.get_namespaced_custom_object(self._group,
+                                                            self._api_version,
+                                                            self._metadata.namespace,
+                                                            self._plural,
+                                                            self._metadata.name)
 
             success = self.check_build_succeeded(bld)
             if success:
-                logger.error('Build finished successfully.')
+                logger.warn('Build finished successfully.')
```

Comment:
why is this message warn if the build was successful?

Reply:
So that users get a confirmation either in the console or in the notebook that their job was launched without having to change the verbosity of the logger.

Reply:
can info be default log-level?